MatterControlLib/ConfigurationPage/PrintLeveling/WizardPages/GetUltraFineBedHeight.cs

```diff
@@ -51,22 +46,5 @@ public class GetUltraFineBedHeight : FindBedHeight
 				probePositionsBeingEditedIndex)
 		{
 		}
-
-		public override void OnDraw(Graphics2D graphics2D)
-		{
-			haveDrawn = true;
-			base.OnDraw(graphics2D);
-		}
-
-		public override void OnLoad(EventArgs args)
-		{
-			// TODO: Why conditional on haveDrawn?
-			if (haveDrawn)
```

Comment:
@larsbrubaker  - if you know anything about why this was here or what it was doing, it would be great to know more. I can't imagine when moving 2+ in z when we've completed a Draw is the right behavior and I don't think this ever happened because we'd invoke the BecomingActive before the widget was onscreen and we only ever checked once. Seems like active but dead code

Reply:
This is to make sure at the conclusion of fine bed height measuring we are not leaving the hotend directly against the bed.

If we are leveling on a Z-Min printer than this is the only code that get the nozzle off the bed at the conclusion of leveling. It may be more logical to put it in LastPageInstructions.cs as an alternative to Home Z-Max behavior

~~~
			if (printer.Settings.GetValue<bool>(SettingsKey.z_homes_to_max))
			{
				printer.Connection.HomeAxis(PrinterConnection.Axis.XYZ);
			}